### PR TITLE
Add image statistics docs

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============
### Scope of the PR

This PR adds the documentation for the up42 image statistics block.

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by teoEnache — APPROVED** [review]
looks good

**Review by perusio — CHANGES_REQUESTED** [review]
It computes the mean on what? On the value of each band? Is unclear to me what it does. Can you elaborate on it please? Thank you.

**Comment by nekhtiari** [comment]
> It computes the mean on what? On the value of each band? Is unclear to me what it does. Can you elaborate on it please? Thank you.

Added. Please check and if it is good so I can merge.

**Review by perusio — CHANGES_REQUESTED** [review]
But does apply across all bands of the source image?

**Comment by nekhtiari** [comment]
> But does apply across all bands of the source image?

Here we only have images with 1 band. Mostly used for weather or climate analysis, specifically will work with our new s5 geotiff block

**Review by seulgi-up42 — APPROVED** [review]
Looks good to me as well! (The only small comment I have is that the images are a bit vague in the block description link you provided but I'm not sure if this is right place to bring that up).

**Comment by nekhtiari** [comment]
> Looks good to me as well! (The only small comment I have is that the images are a bit vague in the block description link you provided but I'm not sure if this is right place to bring that up).

Nice to have your comment. The images will be taken care of later via Niklas. I hope that makes sense.

File: source/up42-blocks/processing/up42-image-statistics.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.. meta::
+   :description: UP42 processing blocks: Apply statistics on stack of raster inputs
+   :keywords: UP42, processing, Statistics, Time series, Analytics, Raster
+
+.. _up42-image-statistics-block:
+
+Image Statistics
```
Comment (markusuwe) [inline source/up42-blocks/processing/up42-image-statistics.rst:7]:
```suggestion
UP42 Image Statistics
```

File: source/up42-blocks/processing/up42-image-statistics.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+================
+
+For more information, please read the `block description <https://marketplace.up42.dev/block/93d73b17-6d9b-46aa-9745-15dd5d8c4be0>`_.
+
+Block type: ``PROCESSING``
+
+Supported parameters
+--------------------
+
+This block applies statistics on a stack of raster inputs. The statistics can be one of the following: ``mean``, ``min``, ``max``, ``std``, ``median``, or ``sum``.
```
Comment (markusuwe) [inline source/up42-blocks/processing/up42-image-statistics.rst:17]:
```suggestion
This block applies statistics on a stack of raster inputs and creates a raster file containing the result. The statistics can be one of the following: ``mean``, ``min``, ``max``, ``std``, ``median``, or ``sum``.
```

File: source/up42-blocks/processing/up42-image-statistics.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
+.. meta::
+   :description: UP42 processing blocks: Apply statistics on stack of raster inputs
+   :keywords: UP42, processing, Statistics, Time series, Analytics, Raster
+
+.. _up42-image-statistics-block:
+
+Image Statistics
+================
+
+For more information, please read the `block description <https://marketplace.up42.dev/block/93d73b17-6d9b-46aa-9745-15dd5d8c4be0>`_.
```
Comment (markusuwe) [inline source/up42-blocks/processing/up42-image-statistics.rst:10]: Is it worthwhile adding a sentence what kind of inputs can be processed?

**Review by markusuwe — APPROVED** [review]
LGTM

File: source/up42-blocks/processing/index.rst [hunk 4]
```diff
@@ -39,4 +39,4 @@ blocks built by UP42.
    up42-ndvithreshold
    up42-terrasar-geotiff-conversion
    up42-waterfinder
-
+   up42-image-statistics
```
Comment (marooned20) [inline source/up42-blocks/processing/index.rst:42]:
```suggestion
   up42-timeseries-image-statistics
```

File: source/up42-blocks/processing/up42-image-statistics.rst (hunk 3 above)
Comment (nekhtiari) [reply source/up42-blocks/processing/up42-image-statistics.rst:10]: added

**Pull request merged**